src/pint/models/timing_model.py

```
@@ -2409,13 +2440,66 @@ def get_prefix_mapping_component(self, prefix):
         return mapping
 
     def match_param_aliases(self, alias):
-        """Return the parameter corresponding to this alias."""
-        for p in self.params:
-            if p == alias:
-                return p
-            if alias in getattr(self, p).aliases:
-                return p
-        raise ValueError("{} is not recognized as a parameter or alias".format(p))
+        """Return the parameter corresponding to this alias.
+
+        Parameter
+        ---------
+        alias: str
+            Alias name.
+
+        Note
+        ----
+        This function only searches the alias in the component level.
+        """
+        pname = self.aliases_map.get(alias, None)
+        # Split the alias prefix, see if it is a perfix alias
+        try:
+            prefix, idx_str, idx = split_prefixed_name(alias)
+        except PrefixError:
+            if pname is not None:
+                par = getattr(self, pname)
+                if par.is_prefix:
+                    raise ValueError(
+                        "Prefix {} maps to mulitple parameters"
+                        ". Please specify the index as well.".format(alias)
+                    )
+            else:
+                # Not a prefix, not an alias
+                return None
```

Please raise an exception if not found, as you do below.